sonar/theme/templates/sonar/footer.html

```diff
@@ -24,8 +25,22 @@
           </form>
         {%- endif %}
       </div>
+      #}
+      <div class="col-12 col-sm-6 py-2 py-sm-5 text-center text-sm-left">
+        <div class="d-flex flex-row">
+          <div>
+              <a href="#">{{_('Help')}}</a>
+          </div>
+          <div class="ml-3">
+              <a href="#">{{_('About')}}</a>
+          </div>
+          <div class="ml-3">
+              <a href="#">{{_('Contact')}}</a>
```

Comment:
link to contact view, what about "About", "Help"?

Reply:
It's just samples to see how links are displayed in footer. 
I can create views for each links, but there's no content to show on this pages at this time.
Is it better to have links without targets or links pointing to an empty page?

Reply:
Ok but it seems you have a view for contact:
https://github.com/rero/sonar/pull/9/files#diff-71846f390683019c64177dcb3c8b53caR29
no?

Reply:
I removed contact view, but I have to keep views.py otherwise sonar templates are not found anymore